### PR TITLE
test: deprecated method changed

### PR DESCRIPTION
**Pull request by rengare**
#### Please provide a brief summary of this pull request.
Fixing unit test by replacing old method name with new one

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done

### CODE REVIEW

File: libs/platform/src/lib/components/menu/menu-item.component.spec.ts [hunk 1]
```diff
@@ -55,7 +55,7 @@ describe('MenuItemComponent', () => {
         TestBed.configureTestingModule({
             imports: [IconModule],
             declarations: [TestComponent, MenuItemComponent],
-            providers: [MenuKeyboardService]
+            providers: [MenuKeyboardService],
```
Comment (InnaAtanasova) [inline libs/platform/src/lib/components/menu/menu-item.component.spec.ts:58]: extra comma

File: libs/platform/src/lib/components/menu/menu-item.component.spec.ts [hunk 2]
```diff
@@ -18,7 +18,7 @@ import { MenuKeyboardService, IconModule } from '@fundamental-ngx/core';
                 (itemClick)="onItemClick($event)"
             ></fdp-menu-item>
         </div>
-    `
+    `,
```
Comment (InnaAtanasova) [inline libs/platform/src/lib/components/menu/menu-item.component.spec.ts:21]: extra comma

**Comment by InnaAtanasova** [comment]
I add the Platform team as reviewers as it touches Platform code

**Comment by rengare** [comment]
Why spec file was deleted?

**Comment by droshev** [comment]
@rengare what do we do with this PR?

**Comment by rengare** [comment]
Spec file is no longer there, I'm closing PR

**Pull request closed without merging**